### PR TITLE
Circle CI: add pycodestyle

### PR DESCRIPTION
**Pull request by ccrisan**

### CODE REVIEW

File: colibris/skeleton/__packagename__/routes.py [hunk 1]
```diff
@@ -8,11 +8,11 @@
 # Here are some examples of routes. Just remove what you don't need.
 
 ROUTES = [
-    (r'/',                      views.HomeView),
-    (r'/health',                views.HealthView),
-    (r'/users/me',              views.MeView),
-    (r'/users',                 views.UsersView),
-    (r'/users/{id:\d+}',        views.UserView)
```
Comment (lucifurtun) [inline colibris/skeleton/__packagename__/routes.py]: Yeey!

File: .circleci/config.yml [hunk 2]
```diff
@@ -23,7 +30,8 @@ workflows:
   version: 2
   tests:
     jobs:
-      - automated-tests
+      - check-code-style
+      - run-automated-tests
```
Comment (lucifurtun) [inline .circleci/config.yml]: The CI won't create a container for each job, right? (2 containers for our case).

**Comment by lucifurtun** [comment]
Looks good!

File: .circleci/config.yml (hunk 2 above)
Comment (ccrisan) [reply .circleci/config.yml]: Actually it will create two containers and run the jobs in parallel.

**Pull request merged**